recipes-bsp/coreboot-utils/amdtool_git.bb

```
@@ -0,0 +1,31 @@
+require coreboot-utils.inc
+
+SUMMARY = " \
+    A utility similar to inteltool, which dumps useful information on AMD CPUs \
+    for porting boards to coreboot. \
+"
+
+LICENSE = "GPL-2.0-or-later"
+LIC_FILES_CHKSUM = "file://LICENSES/GPL-2.0-or-later.txt;md5=261bea1168c0bdfa73232ee90df11eb6"
+
+DEPENDS += "pciutils zlib"
+
+EXTRA_OEMAKE = ' \
+               DESTDIR="${D}" \
+               PREFIX="${prefix}" \
+               '
+
+SRC_URI += " \
```

```suggestion
SRC_URI = " \
```

Otherwise you'll also fetch source from `coreboot-utils.inc`: `git://github.com/Dasharo/coreboot.git;protocol=https;branch=dasharo-4.21\`

---

fixed: https://github.com/zarhus/meta-coreboot/compare/85333a0be110ac699cae079a491d588f2efa19a5..c8254de6ee875ff4d8f7a3c91880c4d1ed9a7c6e